src/Sulu/Bundle/AdminBundle/Resources/js/components/Tooltip/Tooltip.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+import React from 'react';
+import classNames from 'classnames';
+import {action, observable} from 'mobx';
+import {observer} from 'mobx-react';
+import Popover from '../Popover';
+import tooltipStyles from './tooltip.scss';
+import type {ChildrenArray, Element, ElementRef} from 'react';
+
+type Props = {|
+    children: ChildrenArray<Element<*> | false>,
```

Comment:
if we want to allow all valid react children, we should use `Node` like we do in the `Dialog`:
https://github.com/sulu/sulu/blob/62b24decee4b209a1e54590ae18a590a3bdee01a/src/Sulu/Bundle/AdminBundle/Resources/js/components/Dialog/Dialog.js#L17

Reply:
Need to check that. Node was failing strangely for me in flow when I had it right in my mind.

Reply:
```
Cannot create Tooltip element because React.Element [1] is incompatible with Node [2] in property children.
[incompatible-type]

     src/Sulu/Bundle/AdminBundle/Resources/js/components/Tooltip/tests/Tooltip.test.js
      7│ test('The component should render in unfocused state', () => {
      8│     const component = render(
      9│         <Tooltip label="Copy">
 [1] 10│             <button aria-label="Copy" type="button">
     11│                 <Icon name="su-copy" />
     12│             </button>
     13│         </Tooltip>
     14│     );
     15│

     src/Sulu/Bundle/AdminBundle/Resources/js/components/Tooltip/Tooltip.js
 [2] 11│     children: Node,
 ```

Reply:
Okay need to import the correct type 🙈